modules/default/calendar/calendar.js

```diff
@@ -1,5 +1,7 @@
 /* global CalendarUtils */
 
+const moment = require("moment-timezone");
```

Comment:
Can we use `require` here? `calender.js` is loaded by the browser and not by node, right?